### PR TITLE
Add user interface

### PR DESCRIPTION
**Pull request by bengigo**
**Changes made on this branch:**

- Create an `app.rb` file that will serve as console app entry point.
- Define the entry point in a `main.rb` file with a method
- Present the user with a list of performable options
- Get user input to select from the option
- Perform the selected option
  1. List all books
  2. List all people
  3. Create a person -either _student_ or _teacher_ 
  4. Create a book
  5. Create a rental
  6. List all rentals for a given person id.
 - Allow user the quit the app with an input

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+require_relative 'app'
+
+def main
+  app = App.new
+
+  puts '📖 Welcome to School Library 📖'
+  puts ''
+  # app.present_options
+  # app.check_input
```
Comment (julie-ify) [inline main.rb]: - [x] Kindly remove commented code. We can use commit history to find old code if we want.

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    puts 'ID of person'
+    id = gets.chomp.to_i
+    puts "Rentals by PersonID:#{id}"
+    rentals = @rentals.select { |rental| id == rental.person.id }
+    rentals.each do |item|
+      puts "Date: #{item.date},  Book #{item.book.title}, by #{item.book.author}"
+      puts ''
+      run
+    end
+  end
```
Comment (julie-ify) [inline app.rb]: - [x] When I try to list rental for a person, if I select the ID of a person that does not have a rental the app just quits. From the video demo this scenario should be handled as well. You can just show an empty result, like the video, and still maintain the running app. Kindly fix that.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @bengigo,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Bengi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- good job following the instructions of the previous CR.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**